nasa/cumulus
# CUMULUS-2439: Add CMR search client config to CreateReconciliationReport lambda

**Summary:** Summary of changes

Addresses [CUMULUS-2439: create-reconciliation-report not retrieving all expected collections from CMR](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2439)

## Changes

* Added CMR search client setting to the CreateReconciliationReport lambda function.
* Added `cmr_search_client_config` tfvars to the archive and cumulus terraform modules.
* Updated CreateReconciliationReport lambda to search CMR collections with CMRSearchConceptQueue.

## Test:
* Configure cmr_search_client_config in terraform.tfvars, e.g.
cmr_search_client_config = {
  create_reconciliation_report_cmr_limit = 10
  create_reconciliation_report_cmr_page_size = 5
}
* Check environment variable for prefix-CreateReconciliationReport lambda, it should have CMR_LIMIT = 10 and CMR_PAGE_SIZE = 5
* Create an inventory report, and verify that collections only in CMR are listed and > 10.

## PR Checklist
